### PR TITLE
feat(Community Mint Token): Add select wallet account option

### LINKED ISSUE #9842: [Community Mint Token] Add select wallet account option
### Description

[Design](https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=27140%3A521163&t=9YYh8bbYxML7AoXu-1)

Screenshot 2023-03-17 at 13 15 10

### PR DESCRIPTION
**Pull request by noeliaSD**
Closes #9842

### What does the PR do
- It adds accounts selector in `Create new token` page.
- It adds account name box in `Preview` page.
- It updates `storybook` with new account data.
- It adds variant to `StatusComboBox`

### Affected areas
Community Settings / Mint token
StatusQ / StatusComboBox

### Screenshot of functionality
https://user-images.githubusercontent.com/97019400/226570243-3448af51-75f2-4730-a252-f31f99330836.mov

### CODE REVIEW

**Review by caybro — COMMENTED** [review]
Code LGTM, however I'm missing one component :)

File: ui/app/AppLayouts/Chat/views/communities/CommunityCollectibleView.qml [hunk 1]
```diff
@@ -166,6 +170,13 @@ StatusScrollView {
                 leftAlignment: false
             }
 
+            CustomPreviewBox {
```
[the remark this answers is not in the document]
Comment (noeliaSD) [reply ui/app/AppLayouts/Chat/views/communities/CommunityCollectibleView.qml:173]: In `line 111` there is the inline definition for this component and in `line 35` the alias to set the value!

[the remark this answers is not in the document]
Comment (caybro) [reply ui/app/AppLayouts/Chat/views/communities/CommunityCollectibleView.qml:173]: :coffee:

**Review by caybro — APPROVED** [review]
LGTM

File: ui/app/AppLayouts/Chat/views/communities/CommunityNewCollectibleView.qml [hunk 2]
```diff
@@ -125,60 +113,41 @@ StatusScrollView {
...
-            spacing: rowComponent.isSwitchCase ? 64 : 32
+        CustomLabelDescriptionComponent {
+            Layout.topMargin: Style.current.padding
+            label: qsTr("Select account")
+            description: qsTr("The account on which this token will be minted")
+        }
 
-            ColumnLayout {
-                Layout.fillWidth: true
+        StatusComboBox {
```
[the remark this answers is not in the document]
Comment (micieslak) [reply ui/app/AppLayouts/Chat/views/communities/CommunityNewCollectibleView.qml:122]: Probably it's problem of `StatusComboBox` itself but clicking in the upper part of the component doesn't open the popup. Imo the whole component should react on click.

**Review by micieslak — CHANGES_REQUESTED** [review]
Nice work in general, only minor comments and suggestions.

File: ui/app/AppLayouts/Chat/views/communities/CommunityCollectibleView.qml [hunk 3]
```diff
@@ -31,6 +31,8 @@ StatusScrollView {
     property int chainId
     property string chainIcon
     property int deployState
+    property string accountAddress
```
[the remark this answers is not in the document]
Comment (noeliaSD) [reply ui/app/AppLayouts/Chat/views/communities/CommunityCollectibleView.qml:34]: Ok. No need to add a new signal, will create a d property in the panel and once the `mintCollectible`signal is triggered by `CommunityCollectibleView`, propagate the needed information to the next layer.

File: ui/app/AppLayouts/Chat/views/communities/CommunityNewCollectibleView.qml [hunk 4]
```diff
@@ -234,4 +203,128 @@ StatusScrollView {
...
+    component CustomStatusInput: StatusInput {
+        id: customInput
+
+        property string errorText
+
+        Layout.fillWidth: true
+        validators: [
+            StatusMinLengthValidator {
+                minLength: 1
+                errorMessage: Utils.getErrorMessage(root.errors,
```
[the remark this answers is not in the document]
Comment (noeliaSD) [reply ui/app/AppLayouts/Chat/views/communities/CommunityNewCollectibleView.qml:217]: Ups! `customInput` insted!

File: ui/app/AppLayouts/Chat/views/communities/CommunityNewCollectibleView.qml (hunk 2 above)
[the remark this answers is not in the document]
Comment (noeliaSD) [reply ui/app/AppLayouts/Chat/views/communities/CommunityNewCollectibleView.qml:122]: Yes, it is an issue with the `StatusComboBox` but I think that with then new variant. We can check it in a different task:  https://github.com/status-im/status-desktop/issues/9977

**Comment by noeliaSD** [comment]
Pushed updates according to comments. @micieslak please review them again!

**Review by micieslak — CHANGES_REQUESTED** [review]
Looks good but
```CommunityCollectibleViewPage```
fails for me:
```
file:///home/michalc/workspace/status-desktop/storybook/pages/CommunityCollectibleViewPage.qml:41:17: Cannot assign to non-existent property "accountAddress"
```

**Review by micieslak — COMMENTED** [review]
Combo box now is clickable only in the area of the indicator :)

**Comment by noeliaSD** [comment]
> Looks nice in general but `CommunityCollectibleViewPage` fails for me:
> 
> ```
> file:///home/michalc/workspace/status-desktop/storybook/pages/CommunityCollectibleViewPage.qml:41:17: Cannot assign to non-existent property "accountAddress"
> ```

Solved!

**Comment by noeliaSD** [comment]
> Combo box now is clickable only in the area of the indicator :)

It will be reviewed here: https://github.com/status-im/status-desktop/issues/9977

**Review by micieslak — APPROVED** [review]
Nice :+1:

**Pull request merged**